j5/vision/markers.py

```diff
@@ -1,35 +1,59 @@
 """Markers Class."""
 
-from __future__ import annotations
-
+from abc import ABCMeta
 from typing import List, NamedTuple, Union
 
 import numpy as np
 
 
+class Coordinate(metaclass=ABCMeta):
+    """A position in space."""
+
+    _cart: 'Cartesian'
+    _cyl: 'Cylindrical'
+    _sph: 'Spherical'
+
+    def __init__(self, x: float, y: float, z: float):
+        self._cart = Cartesian(x, y, z)
```

Comment:
Could the constructor be able to take cylindrical and spherical coordinates as well as catesian coordinates?